nntrainer/graph/network_graph.cpp

```diff
@@ -842,6 +843,61 @@ NetworkGraph::finalizeContext(const std::shared_ptr<LayerNode> &lnode,
   return outputs;
 }
 
+std::map<std::string, std::vector<unsigned int>>
+NetworkGraph::getLayerExecutionOrders(const std::shared_ptr<LayerNode> &lnode) {
+  const GraphNode &gnode = *lnode.get();
+  auto init_context = lnode->getInitContext();
+  auto out_specs = init_context.getOutSpecs();
```

Comment:
NNtrainer changes the init_context to runContext after initialization. does it work after initialization or it only works after compile? If it is, plz leave the comment and add the error saying it has limitation.

Reply:
As we discussed in offline, I'll extract `init_context` related codes to DEBUG feature.
thanks for checking :+1: